examples/ex5p.cpp

```diff
@@ -260,6 +260,9 @@ int main(int argc, char *argv[])
    }
    else
    {
+      HypreParMatrix matB;
+      bVarf->FormRectangularSystemMatrix(empty_tdof_list, empty_tdof_list, matB);
```

Comment:
@alecampos2009, if I had to guess, I would suspect as issue from ‘bVarf’ being assembled twice: once on this line as part of the call to ‘FormRectangularSystemMatrix’ and again on line 267 that call ‘ParallelAssemble’.

It might be worth commenting out line 267 and replacing ‘matB’ with ‘B’ on this line